lepton/starterkits/non-english-llm-reasoning.ipynb

```diff
@@ -0,0 +1,1035 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Training a non-English Reasoning Language Model on Lepton with NeMo 2.0\n",
+        "\n",
+        "[NVIDIA NeMo Framework](https://docs.nvidia.com/nemo-framework/user-guide/latest/overview.html) is a powerful toolkit for training and fine-tuning Large Language Models. This notebook demonstrates how to create a reasoning-capable LLM for Spanish on Lepton using NeMo 2.0.\n",
+        "\n",
+        "While many state-of-the-art models perform well in English, their performance often degrades significantly in other languages. This tutorial shows how data translation, continued pre-training (CPT), and supervised fine-tuning (SFT) can be combined to create a model that reasons in English and outputs answers in Spanish. This approach maintains strong reasoning capabilities while enabling Spanish language output for better accessibility.\n",
+        "\n",
+        "## Objectives\n",
+        "\n",
+        "This tutorial demonstrates a complete pipeline for training a reasoning-capable Large Language Model that reasons in English and outputs answers in Spanish, with a focus on the math domain:\n",
+        "1. **Data Translation**: We will translate raw texts and instruction-tuning datasets with reasoning traces from English to Spanish\n",
+        "2. **Continued Pre-Training (CPT)**: We will adapt the model to Spanish language and the math domain\n",
+        "3. **Supervised Fine-Tuning (SFT)**: We will teach the model to reason in English and output answers in Spanish in the math domain\n",
+        "\n",
+        "## Workflow Overview\n",
```

Comment:
For more visual differentiation on the platform, can you update this header to level 3 (`###`)?